examples/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:20.04
```

Comment:
Lets move both these files into a different directory, instead of `examples` maybe `etc/docker/elyra-development`?